src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestStruct.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace System.Text.Json.Serialization.Tests
 {
-    public class SimpleTestStruct : ITestClass
+    public struct SimpleTestStruct : ITestClass
```

Comment:
Fyi, @YohDeadfall - I believe this was meant to be a struct as @buyaa-n has discovered.

The code coverage of some of the recent struct support change was low without this test case and there were a couple of edge cases that don't currently work. Could you please take a look at these and provide some guidance on a starting point (in case you won't be able to address them in the next 1-2 weeks). Appreciate the help!
https://github.com/dotnet/corefx/issues/38414
https://github.com/dotnet/corefx/issues/38435

Reply:
Probably this was broken during rebase. Sorry.

Reply:
No worries, I was guessing something like that happened, nothing was broken related to this change

Reply:
> Probably this was broken during rebase. Sorry.

Yep, it happens sometimes. As @buyaa-n mentioned, don't sweat it :)

Running the tests with `/p:CodeCoverage=true` helped detect the issue here.